src/NavBar.tsx

```diff
@@ -6,6 +6,7 @@ const NavBar = () => (
 		<NavLink to="/">Home</NavLink>
 		<NavLink to="/entrees"> Entrees </NavLink>
 		<NavLink to="/appetizers"> Appetizers </NavLink>
+		<NavLink to="/sign-up"> SignUp </NavLink>
```

Comment:
I think Sign Up is 2 words. Or dashed might work. Try googling it

Reply:
You are right its two words. i changed to 'Sign-Up'